src/components/cc-img/cc-img.js

```diff
@@ -35,8 +35,8 @@ export class CcImg extends LitElement {
     /** @type {string|null} Sets `src` attribute on inner native `<img>` element. */
     this.src = null;
 
-    /** @type {string|null} Sets short fallback text to display when the image cannot be loaded or if `src` is not defined and `skeleton` is `false`. */
-    this.text = null;
+    /** @type {string|null} Sets short fallback accessibleName to display when the image cannot be loaded or if `src` is not defined and `skeleton` is `false`. */
```

Comment:
**🤓 nitpick:** Actually, I think it makes more sense to keep the word "text" for this comment.